src/org/zaproxy/zap/extension/websocket/ui/WebSocketUiHelper.java

```diff
@@ -448,6 +452,54 @@ public ZapTextField getPatternTextField() {
 	}
 	
 	public String getPattern() {
-		return patternTextField.getText();
+		if(patternTextField.getText() == null || patternTextField.getText().equals("")) {
+			return null;
+		}else{
+			return patternTextField.getText();
+		}
+	}
+
+	public void setPattern(String pattern){
+        patternTextField.setText(pattern == null ? null : pattern);
```

Comment:
If pattern is null use null else pattern....how about just use pattern? If it's ok to use null (which pattern might be) ;)